pennylane_cirq/cirq_device.py

```diff
@@ -126,18 +116,18 @@ def __init__(self, wires, shots, qubits=None):
         "SWAP": CirqOperation(lambda: cirq.SWAP),
         "CZ": CirqOperation(lambda: cirq.CZ),
         "PhaseShift": CirqOperation(lambda phi: cirq.ZPowGate(exponent=phi / np.pi)),
-        "RX": CirqOperation(lambda phi: cirq.Rx(phi)),
-        "RY": CirqOperation(lambda phi: cirq.Ry(phi)),
-        "RZ": CirqOperation(lambda phi: cirq.Rz(phi)),
-        "Rot": CirqOperation(lambda a, b, c: [cirq.Rz(a), cirq.Ry(b), cirq.Rz(c)]),
-        "CRX": CirqOperation(lambda phi: cirq.ControlledGate(cirq.Rx(phi))),
-        "CRY": CirqOperation(lambda phi: cirq.ControlledGate(cirq.Ry(phi))),
-        "CRZ": CirqOperation(lambda phi: cirq.ControlledGate(cirq.Rz(phi))),
+        "RX": CirqOperation(cirq.rx),
+        "RY": CirqOperation(cirq.ry),
+        "RZ": CirqOperation(cirq.rz),
```

Comment:
I was wondering about these three, how come the codecov had concerns only about these?

Reply:
Because here he could be sure that the lambda was not necessary. For example in `lambda: cirq.SWAP`, `cirq.SWAP` is not callable, but `CirqOperation` expects a callable as argument, so the lambda is necessary.

Reply:
Hmm, that's a fair point, thanks! :)